### PR TITLE
gf180mcu: add gf180mcuD variant for 5LM_1TM_11K

### LINKED ISSUE #342: add gf180mcuD for 5LM + 11K top

### PR DESCRIPTION
**Pull request by proppy**
Fixes #342

### CODE REVIEW

**Review by RTimothyEdwards — APPROVED** [review]
Looks good except for a typo in README, which I will fix after merging and testing.

File: gf180mcu/README [hunk 1]
```diff
@@ -34,7 +34,8 @@ The installation directory below PDK_ROOT is the name of the PDK;  e.g.,
 
 	    gf180mcuA/	3-metal backend stack
 	    gf180mcuB/	4-metal backend stack
-	    gf180mcuC/	5-metal backend stack
+	    gf180mcuC/	5-metal backend stack (9K)
+	    gf180mcuC/	5-metal backend stack (11K)
```
Comment (proppy) [inline gf180mcu/README:38]: ah yes, that should be `D` here.

**Comment by RTimothyEdwards** [comment]
Also "ENABLED_VARIANTS" in gf180mcu/Makefile.in should be "A B C D".  I'll update that as well in just a moment.

**Comment by RTimothyEdwards** [comment]
Pulled, merged, and mirrored!

**Pull request closed without merging**